### ISSUE TITLE
warning on build; can't move the window; drag-drop don't work on linux 

### ISSUE THREAD

**Issue by Inkeliz** [sciter-sdk/go-sciter#131]
I'm testing compiling it on Fedora 27, using the GCC 7.3.1 and also I try with 6.4.1, both have the same problem.

I'm using `GO 1.10.1 linux/amd64`, `GCC 7.3.1` and `GTK3 3.22.26.2`/`GTK3-devel 3.22.26.2`. The Sciter itself seems to be installed correctly, when run `ldconfig -p | grep sciter` results `libsciter-gtk.so (libc6, x86-64) => /home/inkeliz/Documentos/sciter/bin.gtk/x64/libsciter-gtk.so`.

However, when I `go build` I get:

```
../../sciter-sdk/go-sciter/include/tiscript.h:236:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    void     (TISAPI *unpin)(tiscript_pvalue* pp);
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:240:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    tiscript_value (TISAPI *native_function_value)(tiscript_VM* pvm, tiscript_method_def* p_method_def);
    ^~~~~~~~~~~~~~
../../sciter-sdk/go-sciter/include/tiscript.h:241:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    tiscript_value (TISAPI *native_property_value)(tiscript_VM* pvm, tiscript_prop_def* p_prop_def);
    ^~~~~~~~~~~~~~
../../sciter-sdk/go-sciter/include/tiscript.h:247:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    bool           (TISAPI *post)(tiscript_VM* pvm, tiscript_callback* pfunc, void* prm);
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:258:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    bool  (TISAPI *set_remote_std_streams)(tiscript_VM* pvm, tiscript_pvalue* input, tiscript_pvalue* output, tiscript_pvalue* error);
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:261:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    bool  (TISAPI *set_nth_retval)(tiscript_VM* pvm, int n, tiscript_value ns );
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:263:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    int   (TISAPI *get_length)(tiscript_VM* pvm, tiscript_value obj );
    ^~~
../../sciter-sdk/go-sciter/include/tiscript.h:265:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    bool  (TISAPI *get_next)(tiscript_VM* pvm, tiscript_value* obj, tiscript_value* pos, tiscript_value* val);
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:267:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    bool  (TISAPI *get_next_key_value)(tiscript_VM* pvm, tiscript_value* obj, tiscript_value* pos, tiscript_value* key, tiscript_value* val);
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:270:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    bool  (TISAPI *set_extra_data)(tiscript_VM* pvm, void* data);
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:271:4: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
    void* (TISAPI *get_extra_data)(tiscript_VM* pvm);
    ^~~~
../../sciter-sdk/go-sciter/include/tiscript.h:284:1: warning: ‘__cdecl__’ attribute ignored [-Wattributes]
 typedef void EXTAPI  TIScriptLibraryInitFunc(tiscript_VM* vm, tiscript_native_interface* piface );
 ^~~~~~~
sciter-x-api.c: In function ‘SAPI’:
sciter-x-api.c:130:17: warning: implicit declaration of function ‘readlink’; did you mean ‘readdir’? [-Wimplicit-function-declaration]
             if (readlink ("/proc/self/exe", folderpath, sizeof(folderpath)) != -1)
                 ^~~~~~~~
                 readdir
```

_The `warning: ‘__cdecl__’ attribute ignored [-Wattributes]` is repeated many times._

**Comment by pravic** [comment]
> warning: implicit declaration of function ‘readlink’

This is odd, because that `readlink()` is declared in [`unistd.h`](https://linux.die.net/man/2/readlink) which is included in `sciter-x-api.h` and `sciter-x.h`.

Can you check `man 2 readlink` under Fedora?

**Comment by Inkeliz** [comment]
Sorry, I didn't notice it. The problem was `fatal error: runtime: out of memory`, I set to 16GB of RAM, previous 4GB,it was fixed, now I can build! 👍 

However I'm facing two issues:
- I can't move the window, it's stucked in the corner. I'm using `window-frame="solid"` and the `<header role="window-caption">`, it works on Windows.
- I can't drag files to the window. I can't drag one file and drop inside the window. I'm using [this code](https://github.com/sciter-sdk/go-sciter/issues/121#issuecomment-372187097) and it works on Windows.

Also, the button/textarea/input have one border, that don't appear on Windows, I'll check the CSS to force one `border: 0;` or something like that.
